src/analyzer/protocol/krb/krb-analyzer.pac

```diff
@@ -307,27 +306,15 @@ refine connection KRB_Conn += {
 				{
 				switch ( ${msg.safe_body.args[i].seq_meta.index} )
 					{
+					case 0:
+						rv->Assign(3, to_stringval(${msg.safe_body.args[i].args.user_data.encoding.content}));
+						break;
 					case 1:
 						timestamp_i = i;
 						break;
 					case 2:
 						timestamp_usecs = binary_to_int64(${msg.safe_body.args[i].args.usec.encoding.content});
 						break;
-					default:
-						break;
-					}
-				}
-
-			if ( timestamp_i )
-				rv->Assign(4, GetTimeFromAsn1(${msg.safe_body.args[timestamp_i].args.timestamp}, timestamp_usecs));
-
-			for ( uint i = 0; i < ${msg.safe_body.args}->size(); ++i )
```

Comment:
Curios!

Reply:
I have no idea why there were two loops that ran over the same array.